### PR TITLE
drop the (few) lombock usage with something different #276

### PR DESCRIPTION
**Pull request by remmeier**
most recent IntelliJ versions provides similar functionality (code folding) for any Java source file, partly obsoleting the need for lombock (like the getter/setters).

### CODE REVIEW

**Comment by chb0github** [comment]
This seems to mix a few things other than lombok removal. I swore I removed vertex and UI package.

**Comment by remmeier** [comment]
code formatting this fairly broken in the vertx project. ok, if it supposed to get removed, I don't bother anymore and just throw it away with a subsequent PR.

**Comment by remmeier** [comment]
for katharsis-ui I created a new branch with a first working but basic version, so that does not need be deleted.

**Comment by chb0github** [comment]
We should have an issue around UI dev. I am thinking of the easiest/best way to get there.

**Comment by remmeier** [comment]
there is https://github.com/katharsis-project/katharsis-framework/issues/165

File: katharsis-examples/vertx-simple/src/main/java/io/katharsis/vertx/examples/KatharsisVerticle.java [hunk 1]
```diff
@@ -13,63 +13,68 @@
...
-            @Override
-            public void handle(RoutingContext routingContext) {
-                HttpServerResponse response = routingContext.response();
-                response.putHeader("content-type", "text/html")
-                        .end("<h1>Hello from my first Vert.x 3 application</h1>" +
-                                "<a href='/api/projects'>/api/projects</a>");
-            }
-        };
-    }
+    router.route("/api/*").handler(katharsisGlue);
```
Comment (chb0github) [inline katharsis-examples/vertx-simple/src/main/java/io/katharsis/vertx/examples/KatharsisVerticle.java]: This has nothing to do with Lombok. Needs to be removed

File: katharsis-examples/vertx-simple/src/main/java/io/katharsis/vertx/examples/KatharsisVerticle.java [hunk 2]
```diff
@@ -13,63 +13,68 @@
...
-    @Override
-    public void start(final Future<Void> fut) throws Exception {
-        // We don't use default methods to be compatible with Java 7.
-        // Create a router object.
-        final Router router = new RouterImpl(vertx);
+  private Handler<RoutingContext> requestHandler;
 
-        // Bind "/" to our hello message - so we are still compatible.
-        router.route("/").handler(helloHandler());
+  public KatharsisVerticle(Vertx vertx) {
```
Comment (chb0github) [inline katharsis-examples/vertx-simple/src/main/java/io/katharsis/vertx/examples/KatharsisVerticle.java]: This has nothing to do with lombok

File: katharsis-examples/vertx-simple/src/main/java/io/katharsis/vertx/examples/repository/TaskToProjectRepository.java [hunk 3]
```diff
@@ -1,45 +1,48 @@
...
-        log.info("Find one target {} {} {}", sourceId, fieldName, queryParams);
-        return Project.builder().id(sourceId).name("find one target " + fieldName).build();
-    }
-
-    @Override
-    public Iterable<Project> findManyTargets(Long sourceId, String fieldName, QueryParams queryParams) {
-        log.info("Find many targets {} {} {}", sourceId, fieldName, queryParams);
-        return Arrays.asList(Project.builder().id(sourceId).name("find many targets " + fieldName).build());
-    }
+  private Logger log = LoggerFactory.getLogger(TaskToProjectRepository.class);
```
Comment (chb0github) [inline katharsis-examples/vertx-simple/src/main/java/io/katharsis/vertx/examples/repository/TaskToProjectRepository.java]: Although it's inconsequential, remove the formatting changes so the PR can focus on just the changes mentioned.

File: katharsis-vertx/src/main/java/io/katharsis/vertx/KatharsisHandlerFactory.java [hunk 4]
```diff
@@ -26,88 +26,77 @@
 import io.katharsis.resource.registry.ResourceRegistry;
 import io.katharsis.resource.registry.ServiceUrlProvider;
 import io.vertx.core.json.Json;
-import lombok.NonNull;
 
 public class KatharsisHandlerFactory {
 
-    public static KatharsisHandler create(@NonNull String packagesToScan,
-                                          @NonNull String webPath) {
+  public static KatharsisHandler create(String packagesToScan, String webPath) {
```
Comment (chb0github) [inline katharsis-vertx/src/main/java/io/katharsis/vertx/KatharsisHandlerFactory.java]: Please remove formatting changes

**Comment by remmeier** [comment]
well if the project is deleted anyway, there is not really a point fixing anything anymore.

File: katharsis-examples/vertx-simple/src/main/java/io/katharsis/vertx/examples/KatharsisVerticle.java (hunk 2 above)
Comment (remmeier) [inline katharsis-examples/vertx-simple/src/main/java/io/katharsis/vertx/examples/KatharsisVerticle.java]: lombock created that constructor

**Pull request closed without merging**